pep/app/pods/browse/journal/volume/route.ts

```diff
@@ -21,12 +24,24 @@ export default class BrowseJournalVolume extends Route {
         });
     }
 
+    /**
+     * Load the first journal for a specific source for information
+     *
+     * @memberof BrowseJournalIndex
+     */
+    async afterModel() {
+        const journalParams = this.paramsFor('browse.journal') as BrowseJournalParams;
+        const journal = await this.store.query('journal', { sourcecode: journalParams.pep_code });
+        this.journalInformation = journal.firstObject;
+    }
```

Comment:
both `browse.journal.index` and `browse.journal.volume` fetch the journal model, so we should probably instead fetch it as the parent `browse.journal` route's main `model()` and then in both sub routes just use `this.modelFor('browse.journal')` (which you could do directly in setupController since its synchronous)